app/controllers/signup.js

```diff
@@ -259,7 +259,8 @@ export default class SignupController extends Controller {
   @action emailValidator(email) {
     if (typeof email !== 'string') return false;
 
-    const pattern = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+    const pattern =
+      /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```

Comment:
why have you added it in the next line?

Reply:
I am getting continuous integration run failed error because of prettier. That is why I have changed this.
Should i revert this?

Reply:
I don't know why is that happening but I do think that these changes shouldn't be there in this PR. So can you revert it?

Reply:
ok